Makefile

```diff
@@ -160,7 +160,7 @@ ENVTEST ?= $(TESTBIN)/setup-envtest
 envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
 $(ENVTEST): $(TESTBIN)
 ifeq (, $(shell ls $(TESTBIN)/setup-envtest 2>/dev/null))
-	GOBIN=$(TESTBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@c7e1dc9b5302d649d5531e19168dd7ea0013736d
+	GOBIN=$(TESTBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.0.0-20250620151452-b9a9ca01fd37
```

Comment:
this version requires golang 1.24, can we just use release-0.20 which only requires golang 1.23

Reply:
```
go install sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.20.0
go: downloading sigs.k8s.io/controller-runtime v0.20.0
go: sigs.k8s.io/controller-runtime/tools/setup-envtest@v0.20.0: module sigs.k8s.io/controller-runtime@v0.20.0 found, but does not contain package sigs.k8s.io/controller-runtime/tools/setup-envtest
```

in my test ,i can't use 0.20.0 tag for setup-envtest

since we use 1.23+, maybe we should use `latest`, otherwise we should only use commitid
<img width="1041" alt="image" src="https://github.com/user-attachments/assets/2b7448bb-cfcf-4ed7-a416-af21faeaae9f" />